proxy/Cargo.toml

```diff
@@ -1,64 +1,6 @@
-[package]
-name = "proxy"
-version = "0.1.0"
-license = "GPL-3.0-only"
-authors = ["radicle team <dev@radicle.xyz>"]
-edition = "2018"
-description = "Proxy to serve a specialised API to the Upstream UI."
-readme = "https://github.com/radicle-dev/radicle-upstream/blob/master/proxy/README.md"
-categories = [ "crypto", "p2p" ]
-keywords = [ "radicle-upstream", "proxy" ]
-repository = "git@github.com:radicle-dev/radicle-upstream.git"
-default-run = "proxy"
+[workspace]
```

Comment:
How about moving this file to the repository root? That would allow us to run all cargo commands also from the root. Otherwise, nothing changes.

Reply:
The root is already quite crowded. We have three main components to manage: app container, ui and proxy. Not generally opposed to move, just a not convinced that we have quick ways to know what relevance a file has for which of our artefacts.